corehq/motech/repeaters/views/repeaters.py

```diff
@@ -286,6 +286,15 @@ def page_url(self):
         return reverse(AddFormRepeaterView.urlname, args=[self.domain])
 
 
+class AddConnectFormRepeaterView(AddFormRepeaterView):
+    urlname = "add_connect_form_repeater"
+
+    def initialize_repeater(self):
+        repeater = super().initialize_repeater()
+        repeater.add_backoff_code(404)
```

Comment:
Would it make sense to add a comment or maybe a docstring explaining why 404 is being added?